config/v1/types_cluster_version.go

```diff
@@ -806,6 +842,15 @@ type ConditionalUpdate struct {
 // for not recommending a conditional update.
 // +k8s:deepcopy-gen=true
 type ConditionalUpdateRisk struct {
+	// conditions represents the observations of the conditional update
+	// risk's current status. Known types are:
+	// * Apply, for whether the risk is applied to the current cluster.
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
Why 16? It looks like you've only got one known condition type so would a maximum length of 1 be sufficient? Do you anticipate adding additional condition types in the future?